client/data/help/use-help-search-query.ts

```diff
@@ -62,22 +63,30 @@ const fetchArticlesAPI = async (
 				path: `/help-center/articles?${ queryString }`,
 			} as APIFetchOptions ) ) as SearchResult[];
 		}
+
+		if ( articlesResponse?.length >= 0 ) {
+			searchResult = articlesResponse;
+		}
 	}
-	queryString = buildQueryString( { query: search, locale, section: sectionName } );
-	if ( canAccessWpcomApis() ) {
-		searchResultResponse = ( await wpcomRequest( {
-			path: `help/search/wpcom?${ queryString }`,
-			apiNamespace: 'wpcom/v2/',
-			apiVersion: '2',
-		} ) ) as SearchResult[];
-	} else {
-		searchResultResponse = ( await apiFetch( {
-			global: true,
-			path: `/help-center/search?${ queryString }`,
-		} as APIFetchOptions ) ) as SearchResult[];
+
+	// If less than 5 tailored articles are returned, fetch search results.
+	if ( articlesResponse?.length < 5 ) {
+		queryString = buildQueryString( { query: search, locale, section: sectionName } );
+		if ( canAccessWpcomApis() ) {
+			searchResultResponse = ( await wpcomRequest( {
+				path: `help/search/wpcom?${ queryString }`,
+				apiNamespace: 'wpcom/v2/',
+				apiVersion: '2',
+			} ) ) as SearchResult[];
+		} else {
+			searchResultResponse = ( await apiFetch( {
+				global: true,
+				path: `/help-center/search?${ queryString }`,
+			} as APIFetchOptions ) ) as SearchResult[];
+		}
+		// Remove articles that are already in the tailored articles.
+		searchResult = filterOutDuplicatedItems( articlesResponse, searchResultResponse );
```

Comment:
I haven't checked, but can we do 
```
searchResultResponse = filterOutDuplicatedItems( articlesResponse, searchResultResponse );
```
That will result in us needing one less variable